registry-resolver/Cargo.toml

```diff
@@ -14,3 +14,8 @@ serde_json = "1.0.81"
 pbjson = "0.3"
 pbjson-types = "0.3"
 async-trait = "0.1.53"
+mockall = "0.11.1"
+
+[dev-dependencies]
+rstest = "0.15.0"
+async-std = { version = "1.5", features = ["attributes"] }
```

Comment:
we decided on tokio for monetae runtime, would tokio work here as well en lieu of async-std?

Reply:
`rstest` works more easily with `async-std` which is why I migrated to it. Do you prefer tokio over async-std here? If so why?

Reply:
Here's the link to the relevant docs - https://docs.rs/rstest/latest/rstest/attr.rstest.html#async

Reply:
that's reason enough, can we add the hyperlink as a comment above line 21?

Reply:
Done